barretenberg/cpp/pil/vm2/alu.pil

```diff
@@ -1,8 +1,8 @@
 include "constants_gen.pil";
 include "execution.pil";
 include "ff_gt.pil";
+include "gt.pil";
 include "precomputed.pil";
-include "range_check.pil";
 namespace alu;
```

Comment:
sorry for the nit: could we have a newline between the namespace and the includes?